doc/changelog/07-commands-and-options/11828-obligations+depr_hide_obligation.rst

```diff
@@ -0,0 +1,9 @@
+- **Deprecated:**
+  Option :flag:`Hide Obligations` has been deprecated
+  (`#11828 <https://github.com/coq/coq/pull/11828>`_,
+  by Emilio Jesus Gallego Arias).
+
+- **Removed:**
+  Deprecated option :flag:`Shrink Obligations` has been removed
```

Comment:
Since the flag has been removed, you can no longer refer to it like this.

```suggestion
  Deprecated option ``Shrink Obligations`` has been removed
```

There might be other places to fix.

Reply:
Oh indeed, silly me, thanks!